http/vibe/http/server.d

```diff
@@ -1733,7 +1733,7 @@ private void listenHTTPPlain(HTTPServerSettings settings, HTTPServerContext cont
 
 
 private void handleHTTPConnection(TCPConnection connection, HTTPListenInfo listen_info)
-@safe {
+@trusted {
```

Comment:
I know that there are many `@trusted` necessary for Vibe.d to work, but imho it just feels like we lost the battle if we need to put `@trusted` at nearly every function :S